src/styles/App.scss

```diff
@@ -55,7 +55,7 @@
   font-weight: 900;
   position: absolute;
   left: -17px;
-  top: 40px;
+  top: 100px;
   color: #999;
```

Comment:
Colours are likely to be reused. It would be better to use a variable for this. (see https://sass-lang.com/documentation/variables)